Source/WebCore/css/CSSPageRule.cpp

```diff
@@ -56,11 +56,10 @@ CSSStyleDeclaration& CSSPageRule::style()
 String CSSPageRule::selectorText() const
 {
     if (auto* selector = m_pageRule->selector()) {
-        String pageSpecification = selector->selectorText();
-        if (!pageSpecification.isEmpty() && pageSpecification != starAtom())
-            return makeString("@page "_s, pageSpecification);
+        if (!selector->selectorText().isEmpty())
+            return selector->selectorText();
```

Comment:
```suggestion
        return selector->selectorText();
```
This is equivalent, no?

Reply:
It's not necessarily the same if `selectorText()` can return the null string. The code as written will return the non-null empty string if `selectorText()` returns the null string.